lib/components/horizontal_section.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../graphql/schema/pages.graphql.dart';
+import '../graphql/queries/calendar_episode_entries.graphql.dart';
+import '../graphql/queries/page.graphql.dart';
+
+import '../providers/todays_calendar_entries.dart';
+import '../helpers/utils.dart';
+import '../models/analytics/sections.dart';
+import 'horizontal_section_episode.dart';
+import 'horizontal_section_show.dart';
+import 'horizontal_slider.dart';
+import 'section_item_click_wrapper.dart';
+
+const Map<Enum$SectionSize, Size> _defaultImageSize = {
+  Enum$SectionSize.small: Size(140, 80),
+  Enum$SectionSize.medium: Size(240, 146),
+};
+
+const Map<Enum$SectionSize, double> _defaultSliderHeight = {
+  Enum$SectionSize.small: 156,
+  Enum$SectionSize.medium: 222,
+};
+
+const Map<Enum$SectionSize, Size> _posterImageSize = {
+  Enum$SectionSize.small: Size(140, 208),
+  Enum$SectionSize.medium: Size(230, 340),
+};
+
+const Map<Enum$SectionSize, double> _posterSliderHeight = {
+  Enum$SectionSize.small: 284,
+  Enum$SectionSize.medium: 416,
+};
+
+class HorizontalSection extends ConsumerWidget {
```

Comment:
The name clashes with HorizontalSlider and it's not the only horizontally scrollable section (e.g. featured, icon seciton)..
It's more like a style family... ill try to find a name...